.gitignore

```diff
@@ -45,3 +45,4 @@ build-stats.*
 !.yarn/releases
 !.yarn/sdks
 !.yarn/versions
+yarn.lock
```

Comment:
This change is unrelated to this PR, but after thinking about it, it's probably OK to keep since we use Bun for dependency management now anyway.